grails-app/controllers/au/org/emii/portal/LayerController.groovy

```diff
@@ -408,44 +408,17 @@ class LayerController {
                     def xml = new XmlSlurper().parseText(metadataText)
                     //TODO: Validate schema before proceeding
 
-                    //Extract Abstract and resource links
                     def abstractText = HtmlUtils.htmlEscape(xml.identificationInfo.MD_DataIdentification.abstract.CharacterString.text())
-                    def onlineResourcesList = xml.distributionInfo.MD_Distribution.transferOptions.MD_DigitalTransferOptions.onLine.list()
-
-                    def html = "<!DOCTYPE html>\n"
-                    html += "<h4>Abstract</h4>${abstractText}<BR><h4>Online Resources</h4>\n"
-
-                    html += "<ul>\n"
-                    onlineResourcesList.each {
-                        if (!it.CI_OnlineResource.protocol.text().startsWith("OGC:WMS")) {
-                            def linkText = HtmlUtils.htmlEscape(it.CI_OnlineResource.description.CharacterString.text())
-                            def linkProtocol = HtmlUtils.htmlEscape(it.CI_OnlineResource.protocol.CharacterString.text())
-                            def linkUrl = it.CI_OnlineResource.linkage.URL.text()
-                            def linkExternal = ""
-                            if (linkUrl && linkUrl[0] != "/") {
-                                linkExternal = "class=\"external\""
-                            }
-                            // Overcome the case where the URL is valid but has no description
-                            if (!linkText) {
-                                linkText = "<i>Unnamed Resource</i>"
-                            }
-
-                            if (!linkProtocol.startsWith("IMOS:AGGREGATION")) {
-                                html += """<li><a ${linkExternal} href="${linkUrl}" target="_blank">${linkText}</a></li>\n"""
-                            }
-                        }
-                    }
-                    html += "</ul>"
 
-                    responseText = html
+                    responseText = abstractText
                 }
             }
             catch (SAXException e) {
-                log.warn("Error getting formatted metadata, params: ${params}", e)
+                log.warn("Error getting metadata abstract, params: ${params}", e)
                 responseText = "<BR>The metadata record is not available at this time."
```

Comment:
In other words - if the client is already handling errors, and this is just a simple proxy request - let the client do it all the way.